packages/teams-js/src/public/tasks.ts

```diff
@@ -92,7 +91,10 @@ export namespace tasks {
    * @returns - Converted UrlDialogInfo object
    */
   function getUrlDialogInfoFromTaskInfo(taskInfo: TaskInfo): UrlDialogInfo {
-    /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
+    if (taskInfo.url === undefined) {
+      throw new Error("url property of taskInfo object can't be undefined");
+    }
```

Comment:
`TaskInfo` allows the `url` property to be undefined. But if that was ever undefined, this would eventually fail anyway since you need a URL to show a url dialog. This just makes it fail slightly earlier.